{"uuid":"959339f2-6626-43d6-aa94-198e75b7dc29","historyId":"14a58d477d60cf3ad88530c985ecc783","fullName":"com.testing.framework.tests.vwo.TestVWOLogin_01_NormalScript_POM.test_negative_vwo_login","labels":[{"name":"package","value":"com.testing.framework.tests.vwo.TestVWOLogin_01_NormalScript_POM"},{"name":"testClass","value":"com.testing.framework.tests.vwo.TestVWOLogin_01_NormalScript_POM"},{"name":"testMethod","value":"test_negative_vwo_login"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"SeleniumFramework_app.vwo"},{"name":"subSuite","value":"com.testing.framework.tests.vwo.TestVWOLogin_01_NormalScript_POM"},{"name":"host","value":"WS-24701"},{"name":"thread","value":"36896@WS-24701.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"test_negative_vwo_login","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Cannot invoke \"java.util.Properties.getProperty(String)\" because \"p\" is null","trace":"java.lang.NullPointerException: Cannot invoke \"java.util.Properties.getProperty(String)\" because \"p\" is null\r\n\tat com.testing.framework.utils.PropertiesReader.readKey(PropertiesReader.java:25)\r\n\tat com.testing.framework.driver.DriverManager.init(DriverManager.java:26)\r\n\tat com.testing.framework.base.CommonToAllTest.setUp(CommonToAllTest.java:12)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)\r\n\tat org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:810)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:577)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:848)\r\n\tat org.testng.TestRunner.run(TestRunner.java:621)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\r\n\tat org.testng.TestNG.run(TestNG.java:1082)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"Verify that with the invalid email, password error message is shown on the app.vwo.com","steps":[],"attachments":[],"parameters":[],"start":1748525801619,"stop":1748525801619}